bolt04/enctlvs.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "test name": "Simple enctlv for Alice, next is Bob",
+    "node_privkey": "414141414141414141414141414141414141414141414141414141414141414101",
+    "node_id": "02eec7245d6b7d2ccb30380bfbe2a3648cd7a942653f5aa340edcea1f283686619",
+    "blinding_secret": "050505050505050505050505050505050505050505050505050505050505050501",
+    "blinding": "0362c0a046dacce86ddd0343c6d3c7c79c2208ba0d9c9cf24a6d046d21d21f90f7",
+    "encmsg": {
+      "next_node_id": "0324653eac434488002cc06bbfb7f10fe18991e35f9fe4302dbea6d2353dc0ab1c"
+    },
+    "ss": "2e83e9bc7821d3f6cec7301fa8493aee407557624fb5745bede9084852430e3f",
+    "HMAC256('blinded_node_id', ss)": "7d846b3445621d49a665e5698c52141e9dda8fa2fe0c3da7e0f9008ccc588a38",
+    "blinded_node_id": "02004b5662061e9db495a6ad112b6c4eba228a079e8e304d9df50d61043acbc014",
+    "E": "0362c0a046dacce86ddd0343c6d3c7c79c2208ba0d9c9cf24a6d046d21d21f90f7",
+    "H(E || ss)": "bae3d9ea2b06efd1b7b9b49b6cdcaad0e789474a6939ffa54ff5ec9224d5b76c",
+    "next_e": "76d4de6c329c79623842dcf8f8eaee90c9742df1b5231f5350df4a231d16ebcf01",
+    "encmsg_hex": "04210324653eac434488002cc06bbfb7f10fe18991e35f9fe4302dbea6d2353dc0ab1c",
+    "rho": "030ddf4da23c97ada489fbdc4342e701de57f6170e46c66ea6c7585b0933db80",
+    "enctlv_hex": "6970e870b473ddbc27e3098bfa45bb1aa54f1f637f803d957e6271d8ffeba89da2665d62123763d9b634e30714144a1c165ac9"
+  },
+  {
+    "test name": "Blinding-key-override enctlv for Bob, next is Carol",
+    "node_privkey": "424242424242424242424242424242424242424242424242424242424242424201",
+    "node_id": "0324653eac434488002cc06bbfb7f10fe18991e35f9fe4302dbea6d2353dc0ab1c",
+    "blinding_secret": "76d4de6c329c79623842dcf8f8eaee90c9742df1b5231f5350df4a231d16ebcf01",
+    "blinding": "03fc5e56da97b462744c9a6b0ba9d5b3ffbfb1a08367af9cc6ea5ae03c79a78eec",
+    "encmsg": {
+      "next_node_id": "027f31ebc5462c1fdce1b737ecff52d37d75dea43ce11c74d25aa297165faa2007",
+      "blinding": "070707070707070707070707070707070707070707070707070707070707070701"
+    },
+    "ss": "f18a1ddb1cb27d8fc4faf2cf317e87524fcc6b7f053496d95bf6e6809d09851e",
+    "HMAC256('blinded_node_id', ss)": "8074773a3745818b0d97dd875023486cc35e7afd95f5e9ec1363f517979e8373",
+    "blinded_node_id": "026ea8e36f78e038c659beba9229699796127471d9c7a24a0308533371fd63ad48",
+    "E": "03fc5e56da97b462744c9a6b0ba9d5b3ffbfb1a08367af9cc6ea5ae03c79a78eec",
+    "H(E || ss)": "9afb8b2ebc174dcf9e270be24771da7796542398d29d4ff6a4e7b6b4b9205cfe",
+    "next_e": "5149cb0ce6100209f9a2a1140c4e6e8235381aac6201da2c0f8ce1f91bcc3b6e01",
+    "encmsg_hex": "0421027f31ebc5462c1fdce1b737ecff52d37d75dea43ce11c74d25aa297165faa20070c2102989c0b76cb563971fdc9bef31ec06c3560f3249d6ee9e5d83c57625596e05f6f",
```

Comment:
Believe this encodes `next_blinding_override` as TLV type 12, but the TLV type in the spec is 8